internal/integration/provision/upgrade.go

```diff
@@ -70,13 +68,13 @@ type upgradeSpec struct {
 }
 
 const (
-	previousRelease = "v0.8.5"
-	stableRelease   = "v0.9.1" // or soon-to-be-stable
+	previousRelease = "v0.9.1"
+	stableRelease   = "v0.10.0-beta.1" // or soon-to-be-stable
 	// The current version (the one being built on CI) is DefaultSettings.CurrentVersion.
 
-	previousK8sVersion = "1.20.1" // constants.DefaultKubernetesVersion in the previousRelease
-	stableK8sVersion   = "1.20.5" // constants.DefaultKubernetesVersion in the stableRelease
-	currentK8sVersion  = "1.21.0" // next k8s version being tested
+	previousK8sVersion = "1.20.2"      // constants.DefaultKubernetesVersion in the previousRelease
+	stableK8sVersion   = "1.21.0-rc.0" // constants.DefaultKubernetesVersion in the stableRelease
```

Comment:
It will be 1.21.0 release as my PR as my PR is merged

Reply:
alpha.2 is 1.21.0-rc.0 (we got it released yesterday), once we have beta, we'll update this

Reply:
oh I realized I somehow typed 0.10-beta.1, that was my brain twist